### ISSUE TITLE
replaceSingleQuotes undefined check

### ISSUE THREAD

**Issue by olivierlabonte** [hiddentao/squel#271]
https://github.com/hiddentao/squel/blob/86019b242eb46fc752df473076ae7bfabdbf5a8f/dist/squel.js#L613

I have been having problems where activating the replaceSingleQuotes option throws an exception when passing undefined values as parameters. After checking the source code, I noticed there is no check for an undefined value at the line linked above. Is this intended behaviour? I have to manually put null instead of undefined when passing values to work around this issue.

**Comment by hiddentao** [comment]
Yes this is intended behaviour. What are you trying to accomplish?
